src/main/java/com/jwt/jwt/JwtConfig/JwtAuthenticationEntryPoint.java

```diff
@@ -0,0 +1,24 @@
+package com.jwt.jwt.JwtConfig;
+
+import jakarta.servlet.ServletException;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import org.springframework.security.core.AuthenticationException;
+import org.springframework.security.web.AuthenticationEntryPoint;
+import org.springframework.stereotype.Component;
+
+import java.io.IOException;
+import java.io.PrintWriter;
+import java.io.Serializable;
+@Component
+public class JwtAuthenticationEntryPoint implements AuthenticationEntryPoint{
+
+    @Override
+    public void commence(HttpServletRequest request, HttpServletResponse response, AuthenticationException authException) throws IOException, ServletException {
```

Comment:
Since we are just generating token from this app, do we need this?
What is the use of this?

Reply:
I forget to remove this. I will remove it.